### ISSUE TITLE
FMF Mod

### ISSUE THREAD

**Issue by StrawRetro** [luckydog7/Funkin-android#864]
![Screenshot_20210518-165015_YouTube](https://user-images.githubusercontent.com/82880633/118663225-52462300-b7f9-11eb-9c96-d706d4163f30.png)

FMF Mod

**Comment by kffwlp** [comment]
FMF mod

**Comment by AnthonyCTM** [comment]
Yeah i saw it and downloaded it to test it and it is really good, but probably is another ripoff of the luckydog's engine, but it still good

**Comment by AnthonyCTM** [comment]
Yeah i saw it and downloaded it to test it and it is really good, but probably is another ripoff of the luckydog's engine, but it still good

**Comment by georgechen1236** [comment]
I know I download it but is not good

**Comment by AnthonyCTM** [comment]
> I know I download it but is not good

Well the problem are the ads everywere but it haves a good version of midfight masses and i've been looking for that for a long time

**Comment by georgechen1236** [comment]
> > I know I download it but is not good
> 
> Well the problem are the ads everywere but it haves a good version of midfight masses and i've been looking for that for a long time

Whatever

**Comment by arrivedercihehe** [comment]
Friday might funkin
